verify-sna/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "verify-sna",
+  "version": "1.0.0",
+  "private": true,
+  "dependencies": {
+    "sqlite3": "5.0.10",
+    "twilio": "^3.61.0"
+  },
+  "devDependencies": {

```

Comment:
Please remove this section and next ones. jest is already included in the project and next dependency and configurations are included also